test/logger.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env mocha */
+'use strict';
+const Winston = require('winston');
+
+describe('logging', () => {
+  it('returns a WINSTON object', () => {
+    const config = require('../lib/config');
+    const log = require('../lib/logger').attach(config);
+
+    log.should.be.an.instanceOf(Winston.Logger);
```

Comment:
I feel like we shouldn't care what logging library we use, as long as it has the functionality we expect. The tests describe that functionality.

This may be something where we want to add tests to cover functionality as we use it, rather than trying to think through all the possible use case of a logging library up front.